dev/src/platform.cpp

```diff
@@ -197,9 +197,7 @@ bool InitPlatform(const char *exefilepath, const char *auxfilepath, bool from_bu
             write_dir = exepath;
         }
         data_dirs.push_back(string(exepath));
-        #ifdef __linux__
-            data_dirs.push_back("/usr/share/lobster/");
-        #endif
+        data_dirs.push_back(DATADIR);
```

Comment:
Please surround with `#ifdef DATADIR`, this is built on many platforms with many build systems, not just CMake

Reply:
Done.